src/utils/index.ts

```diff
@@ -24,4 +8,4 @@ export async function getAvailablePort(): Promise<number> {
   }
 
   return ports[0];
-}
+}
```

Comment:
```suggestion
}

```
it's stronger than me :see_no_evil:

Reply:
kkkkkkkk